src/Source/WebformComponentSum.php

```diff
@@ -68,6 +76,41 @@ public function widgetForm($item) {
       '#desription' => t('All values with this form key are summed up to get the current value for the progress bar.'),
       '#default_value' => $source_options['form_key'],
     );
+    // add_nids
+    $source_options = isset($item['options']['source']) ? $item['options']['source'] : array();
+    $source_options += array('add_nids' => '');
```

Comment:
Please add all the default config in one place at the beginning of the function. You might want to use `\Drupal\little_helpers\ArrayConfig::mergeDefaults()` to pack this into a single statement.